metpy/calc/thermo.py

```diff
@@ -273,6 +273,8 @@ def el(pressure, temperature, dewpt):
     """
     ideal_profile = parcel_profile(pressure, temperature[0], dewpt[0]).to('degC')
     x, y = find_intersections(pressure[1:], ideal_profile[1:], temperature[1:])
+    # Find LFC to make sure that the function isn't passing off the LFC as the EL
+    LFC_p = lfc(pressure, temperature, dewpt)[0]
```

Comment:
Will want to avoid caps and use descriptive name. `lfc_pressure, _ = lfc(pressure, temperature, dewpt)` That also avoids index into the results at the end of the call.